unity/Assets/Scripts/ARFlowDeviceSample.cs

```diff
@@ -24,15 +25,16 @@ public class ARFlowDeviceSample : MonoBehaviour
     private Vector2Int _sampleSize;
     private bool _enabled = false;
 
+    public TMP_InputField inputField;
+
     // Start is called before the first frame update
     void Start()
     {
         // const string serverURL = "http://192.168.1.100:8500";
         // const string serverURL = "http://169.254.189.74:8500";
         // const string serverURL = "http://192.168.1.139:8500";
-        // const string serverURL = "http://192.168.1.100:8500";
-        const string serverURL = "http://192.168.1.26:8500";
-        _client = new ARFlowClient(serverURL);
+        const string serverURL = "http://192.168.1.219:8500";
+        inputField.text = serverURL;
```

Comment:
Can we make this a bit more user-friendly? I think we should reduce the number of characters users need to input. For example, we could get right off the `http://` part. And can we use two input fields? One for the IP address and another one for the port number. What do you think?

Reply:
Sounds good! I'm on it.